docs/install-mlrun-ce/kubernetes-install.md

```diff
@@ -0,0 +1,262 @@
+- `<your-email>` is your Docker email.
+
+To install the chart with the release name `mlrun-ce` use the following command.  
+Note the reference to the pre-created `registry-credentials` secret in `global.registry.secretName`:
+
+```bash
+helm --namespace mlrun \
+    install mlrun-ce \
+    --wait \
+    --timeout 960s \
+    --set global.registry.url=<registry-url> \
+    --set global.registry.secretName=<registry-credentials> \
+    --set global.externalHostAddress=<host-machine-address> \
+    mlrun-ce/mlrun-ce
+```
+
+Where:
+ - `<registry-url>` is the Registry URL that can be authenticated by the `<registry-credentials>` secret (e.g., `index.docker.io/<your-username>` for Docker Hub).
+ - `<host-machine-address>` is the IP address of the host machine (or `$(minikube ip)` if using minikube).
+
+When the installation is complete, the helm command prints the URLs and ports of all the MLRun CE services.
+
+```{admonition} Known issue when installing the chart on Macs using Apple silicon (ARM-based architicture):
+- The Grafana statistics do not work well in this release. A fix will be delivered in a subsequent release.
+```
+
+## Usage
+
+Your applications are now available in your local browser:
+- Jupyter Notebook - `http://<host-machine-address>:30040`
+- Nuclio - `http://<host-machine-address>:30050`
+- MLRun UI - `http://<host-machine-address>:30060`
+- MLRun API (external) - `http://<host-machine-address>:30070`
+- MinIO API - `http://<host-machine-address>:30080`
+- MinIO UI - `http://<host-machine-address>:30090`
+- Pipeline UI - `http://<host-machine-address>:30100`
+- Grafana UI - `http://<host-machine-address>:30110`
+
+
+```{admonition} Check state
+You can check the current state of the installation via the command `kubectl -n mlrun get pods`, where the main information
+is in columns `Ready` and `State`. If all images have already been pulled locally, typically it takes 
+a minute for all services to start.
+```
+
+```{admonition} Note
+You can change the ports by providing values to the helm install command.
+You can add and configure a Kubernetes ingress-controller for better security and control over external access.
+```
+
+## Start working
+    
+Open the Jupyter notebook on [**jupyter-notebook UI**](http://localhost:30040) and run the code in the 
+[**examples/mlrun_basics.ipynb**](https://github.com/mlrun/mlrun/blob/master/examples/mlrun_basics.ipynb) notebook.
+
+```{admonition} Important
+Make sure to save your changes in the `data` folder within the Jupyter Lab. The root folder and any other folders do not retain the changes when you restart the Jupyter Lab.
+```
+
+## Configuring the remote environment
+
+You can use your code on a local machine while running your functions on a remote cluster. Refer to [Set up your environment](../setup-guide.md) for more information.
+
+### Installing on Docker Desktop
+
+If you are using Docker Desktop, you can install MLRun CE on your local machine.
```

Comment:
This was also mentioned previously (that docker-desktop is the recommended approach). Maybe that previous note should point to this section?